cram_3d_world/cram_bullet_reasoning/src/robot-model.lisp

```diff
@@ -682,3 +682,4 @@ Only one joint state changes in this situation, so only one joint state is updat
                      (physics-utils:resize-3d-model model-part (cl-urdf:size mesh)))
                    model))
           (t model))))
+
```

Comment:
Please pay attention that you don't commit changes like this.
It takes extra time and confuses the person who reviews such a PR, because they think there have been changes in the file but actually there weren't.